docker-compose.yml

```diff
@@ -1,10 +1,11 @@
 version: '3'
 
 services:
-  nodejs:
-    image: rapidpro-cht:1.0.0
-    container_name: rapidpro-cht
+  rapidpro2pg:
+    image: rapidpro2pg:latest
+    container_name: rapidpro2pg
     environment:
-      - pgUrl=postgres://couch2pg:secret=@localhost:5432/db'
-      - rapidProAuth=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
-    network_mode: "host"
\ No newline at end of file
+      - POSTGRESQL_URL=***************************
+      - RAPIDPRO_URL=***************************
+      - RAPIDPRO_AUTH=***************************
```

Comment:
I think the idea here was to allow running as in a docker container, instead of testing in a docker container. 
I did include instruction to test against a dockerised postgres in the readme.